lib/packagers/npm.js

```diff
@@ -103,7 +103,10 @@ class NPM {
     return lockfile;
   }
 
-  static install(cwd) {
+  static install(cwd, packagerOptions) {
+    if (packagerOptions.noInstall) {
+      return BbPromise.resolve();
+    }
```

Comment:
```suggestion
    }

```

This is really nitpicking but consistent with `lib/packagers/yarn.js` below.